example/Sampling/Program.cs

```diff
@@ -33,9 +36,7 @@ public static SampleActivity<ActivityContext> Create(ulong interval)
         
         return (ref ActivityCreationOptions<ActivityContext> options) =>
         {
-            // The string comparison is required here because of some inconsistencies in how empty/default span ids
-            // are constructed.
-            if (options.Parent.SpanId.ToHexString() != default(ActivitySpanId).ToHexString())
+            if (options.Parent != default)
```

Comment:
The condition that necessitates this check only shows up via `Activity.SetParentId()`, which won't affect code on this path.